src/nft/components/collection/CollectionStats.css.ts

```diff
@@ -113,3 +114,65 @@ export const statsValue = style([
     lineHeight: '24px',
   },
 ])
+
+export const statsValueLoading = style([
+  loadingAsset,
+  sprinkles({
+    width: '60',
+    height: '20',
+    marginTop: '8',
+  }),
+])
+
+export const statsLabelLoading = style([
+  loadingAsset,
+  sprinkles({
+    width: '60',
+    height: '16',
+  }),
+])
+
+export const descriptionLoading = style([
+  loadingAsset,
+  sprinkles({
+    fontSize: '16',
+  }),
+  {
+    maxWidth: 'min(calc(100% - 112px), 600px)',
+  },
+])
+
+export const collectionImageIsLoadingBackground = style([
+  collectionImage,
+  sprinkles({
+    backgroundColor: 'backgroundSurface',
+  }),
+])
+
+export const collectionImageIsLoading = style([
+  loadingBlock,
+  collectionImage,
+  sprinkles({
+    borderStyle: 'solid',
+    borderWidth: '4px',
+    borderColor: 'backgroundSurface',
+  }),
+])
+
+export const nameTextLoading = style([
+  loadingAsset,
+  sprinkles({
+    height: '32',
+  }),
+  {
+    width: 236,
+  },
+])
+
+export const iconsLoading = style([
```

Comment:
there might be some flex shrinking going on as the icon widths seem very squished